File: iniciante/javascript/b1008.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let number = parseInt(lines.shift());
let hours = parseInt(lines.shift());
let value = parseFloat(lines.shift());

let salary = hours * value;

console.log('NUMBER = ', number);
console.log('SALARY = U$ ', salary.toFixed(2));

File: iniciante/swift/b1008.swift
import Foundation

let number = Int(readLine(strippingNewline: true)!)!
let hours = Double(readLine(strippingNewline: true)!)!
let valueHours = Double(readLine(strippingNewline: true)!)!

var salary = hours * valueHours 

print("NUMBER = \(number)")
print(String(format:"SALARY = U$ %.2f", salary))